inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/utils/InitializerSpec.kt

```diff
@@ -80,4 +81,19 @@ class InitializerSpec : BaseTest() {
                 .resources
         Initializer.initializeSdk(context, "test", "", true)
     }
+
+    @Test(expected = KeyStoreException::class)
+    fun `should throw keystore exception with null android ID`() {
+        val appCtx = ApplicationProvider.getApplicationContext<Context>()
+        val context = Mockito.mock(Context::class.java)
+        Settings.Secure.putString(appCtx.contentResolver, Settings.Secure.ANDROID_ID, null)
+        When calling context.packageName itReturns appCtx.packageName
+        When calling context.contentResolver itReturns appCtx.contentResolver
+        When calling context.packageManager itReturns appCtx.packageManager
+        When calling context.resources itReturns appCtx.resources
+
+        // AndroidKeyStore is not supported by robolectric
```

Comment:
So you're checking `getUuid()` was called, because it has a side-effect of generating an exception when running as a unit test?

Reply:
yes. I was able to test if it is working fine on the sample app by forcing ANDROID_ID to return null.
There is a workaround to make the keystore work with Robolectric but it is too complicated and not worth the time.